.travis.yml

```diff
@@ -20,7 +20,7 @@ script:
   - make install-php
   - make ci-with-coverage
   - make install-php COMPOSER_FLAGS="--no-dev -q" # Remove dev dependencies to make sure PHPStan creates errors if prod code depends on dev classes
-  - docker run -v $PWD:/app --rm phpstan/phpstan analyse -c phpstan.travis.neon --level 7 --no-progress src/ # Can't use "make stan" because stan was removed
+  - docker run -v $PWD:/app --rm phpstan/phpstan analyse --level 7 --no-progress src/ # Can't use "make stan" because stan was removed
```

Comment:
Sorry to bother you with this again - The Travis output shows a warning when using this image, probably because they no longer host the image on docker hub. We should use our own image here to be consistent.